### ISSUE TITLE
Wrong parameter in Set-DiskPartition.ps1

### ISSUE THREAD

**Issue by bentschrader** [BladeFireLight/WindowsImageTools#2]
I don't know if Microsoft did change the parameter set, but in the Script **Set-Diskpartition.ps1** at line 432 the parameter in _ArgumentList_ called `/setosimage` should be `/setreimage`.
Checked the behaviour on Windows 10 (10.0.17134) and Windows Server 2016 (10.0.14393).
Regards Bent

**Comment by BladeFireLight** [comment]
yes. Microsoft changed thing up a little but and I did not test recovery fully. 
I ahve that fixed and am working to address recoery images. Seems the ISO setup.exe does a poor job on that too. 
If you want to use bitlocker and still be able to reset the OS, you have to folow the docs for win10 but use a larger recovery partition size
